### DISCUSSION TITLE
PQCodeBugetGBRatio - ratio of memory or disk size ?

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [milvus-io/milvus#28584] [Q&A and General discussion] [answered]
Hi guys.
I have a question about the PQCodeBugetGBRatio parameter of the disk index.
In the documentation, it is written that - "Size limit on the PQ code.
A larger value offers a higher recall rate but increases memory usage."
Could you be more precise about what that means? Is it the ratio of in-memory consumption or the ratio of raw data compression?

**Comment by yhmo** [comment, marked as answer]
Disk index uses the PQ algorithm to quantize vectors and keeps the quantizer vectors in memory. PQCodeBugetGBRatio is a ratio of the size of the quantizer vectors vs the size of the original data.
The range is (0.0, 0.25], the default value is 0.125. If it is 0.125, the quantizer vectors size is 1/8 of the original data size.
